azurelinuxagent/agent.py

```diff
@@ -197,45 +217,58 @@ def parse_args(sys_args): # pylint: disable=R0912
     verbose = False
     debug = False
     conf_file_path = None
-    for a in sys_args: # pylint: disable=C0103
-        m = re.match("^(?:[-/]*)configuration-path:([\w/\.\-_]+)", a) # pylint: disable=W1401,C0103
+    log_collector_mode = None
+
+    for arg in sys_args: # pylint: disable=C0103
```

Comment:
I believe the `C0103` error suppression was because the variable name (`a`) doesn't conform to PEP. I think we can remove it now!

Reply:
Nice, will do!

Reply:
I realize this was already resolved, but it looks like the linter error is still being suppressed. Was this maybe a merge error?

Reply:
Ah, could be, let me remove it.